packages/angular/test/base/e2e/src/standalone/tabs.spec.ts

```diff
@@ -0,0 +1,11 @@
+describe('Tabs', () => {
+  beforeEach(() => {
+    cy.visit('/standalone/tabs');
+  });
+
+  it('should render the active tab', () => {
+    cy.get('#tab-button-tab-one').click();
```

Comment:
We should check that the tab is not visible prior to clicking the button for better regression testing around tab activation.

Reply:
Actually.. I was thinking on this more. We should probably have a default route in Angular to redirect to the first tab by default. Loading an empty tabs component isn't a valid use case we should test against. 

Instead we should default to the first tab and then click the `tab-two` button and validate the tab switches.